app/models/user.rb

```diff
@@ -0,0 +1,11 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+
+  has_many :foods, foreign_key: 'user_id'
+  has_many :recipes
+
+  # validates :name, presence: true
```

Comment:
### [OPTIONAL]
It's a good practice to validate a model but in case if you don't want to validate the model then kindly remove the commented code to make the file look clean and professional. 👍